Makefile

```diff
@@ -218,9 +218,17 @@ libzauth:
 #
 # Run this again after changes to libraries or dependencies.
 .PHONY: hie.yaml
-hie.yaml:
+hie.yaml: stack-dev.yaml
 	stack build implicit-hie
-	stack exec gen-hie > hie.yaml
+	stack exec gen-hie > hie.orig.yaml
+	nix-shell --command 'cat hie.orig.yaml | yq "{cradle: {stack: {stackYaml: \"./stack-dev.yaml\", components: .cradle.stack}}}" > hie.yaml'
```

Comment:
Why not simply
```
stack exec gen-hie | nix-shell --run "yq '...'" > hie.yaml
```
?

Reply:
simpler is better, added your suggestion